### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by sgraf812** [atom-haskell/ide-haskell-hasktags#6]
The plugin mostly works on my MacBook (OS X 10.11), but an Atom Helper process sucks a lot of system time.  I tied it to `ide-haskell-hasktags` by disabling it.

Sorry for not providing more information for reproduction, I'll maybe add something later.

**Comment by lierdakil** [comment]
I'm terribly sorry about late response, I read this on a train and then promptly forgot about it completely until today.

`ide-haskell-hasktags` automatically re-runs `hasktags` if any haskell files are changed in project directory. To facilitate that, it relies on `chokidar`. On OSX this boils down to subscribing to `FSEvents`. While initial directory traversal can take up a considerable amount of resources, it usually dials down a lot after that. On larger directories, I think it might take a while though (e.g. if you open a Haskell file from home directory)

If it doesn't sound like I'm describing your particular case, then I have no idea what goes wrong for you, or even if it's a problem related to your particular configuration (since I don't have a spare Mac to test on...)

TL; DR: Check if this problem persists on really small projects (like one Haskell file in a directory or something like that).

**Comment by sgraf812** [comment]
Sorry for procrastination.

I tried it on a `stack new tmp` project and have not encountered the issue so far, so not sure about reproducibility. I'll tell you if I hit this again.
